app/services/data/get-advance-claims-over-specified-date-and-claim-expense-amount.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+module.exports = function (date, status) {
+  // Get all Claims after a certain date or above away with total amount of expenses
```

Comment:
Should this comment be before a certain date instead of after, based on the andWhere below?